extensions-core/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/taskadapter/K8sTaskAdapterTest.java

```diff
@@ -138,12 +138,12 @@ public PodSpec getSpec()
       }
     };
 
-    KubernetesTaskRunnerConfig config = KubernetesTaskRunnerConfig.builder()
-        .withNamespace("test")
-        .withOverlordNamespace("test_different")
-        .withAnnotations(ImmutableMap.of("annotation_key", "annotation_value"))
-        .withLabels(ImmutableMap.of("label_key", "label_value"))
-        .build();
+    KubernetesTaskRunnerStaticConfig config = KubernetesTaskRunnerStaticConfig.builder()
```

Comment:
I think we can still keep the builder method in the interface, so that there will not be any changes in the test cases